src/lib/markdown/markdownToPlainText.ts

```
@@ -0,0 +1,46 @@
+    // without adding whitespace around their content.
+    .replace(/<\/(?:p|div|h[1-6]|li|blockquote|td|th|tr)>|<(?:br|hr)\s*\/?>/g, ' ')
+    .replace(/<[^>]+>/g, '')
+    .replace(/&(?:amp|lt|gt|quot|#39);/g, entity => HTML_ENTITIES[entity])
+    .replace(/\s+/g, ' ')
+    .trim()
+
+  if (options.maxLength === undefined || text.length <= options.maxLength) {
+    return text
+  }
+
+  return `${text.slice(0, options.maxLength - 1).replace(/\s*\S*$/, '')}…`
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)